app/assets/stylesheets/limber/screen.scss

```diff
@@ -122,10 +122,6 @@ header.limber-header {
   }
 }
 
-#app {
```

Comment:
I would be careful changing this. I think some pages require extra space at the bottom. See screenshot - were options are sinking into footer. Maybe they require a fix instead.
![Screenshot 2024-01-15 at 14 59 11](https://github.com/sanger/limber/assets/54811755/4b63544b-3d9f-4472-a3bd-e1b3023da085)